package.json

```diff
@@ -13,7 +13,9 @@
     "precommit": "lint-staged",
     "commit": "git-cz",
     "commitmsg": "validate-commit-msg",
-    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
+    "semantic-release": "semantic-release pre && npm publish && semantic-release post",
+    "snyk-protect": "snyk protect",
+    "prepublish": "npm run snyk-protect"
```

Comment:
To prevent this running on every `install` this should use the `prepublishOnly`, unless that was your intent?

Reply:
Whose intent? You trying to chat with the bot? 😄

Reply:
Not sure this is really the right step for us, anyway. Because we're using semantic-release, the publish step runs on CI after all the work has finished—it's a bit late to be attempting to apply patches 🤔

Reply:
Ohh, just figured this out. I think they're actually taking advantage of this npm quirk as a way to ensure that a local npm install will also run `snyk protect`. Problem is that, since this is a library, this will also run on CI during the publish step.